appkeeper/README.md

```diff
@@ -6,7 +6,22 @@ SIOS AppKeeper automatically restarts failed Amazon EC2 services when notificati
 
 ## Setup
 
-### Get the SIOS AppKeeper API key
+### Agent Installation
+
+The AppKeeper check is not included in the [Datadog Agent][14] package, so you need to install it yourself.
+See [the official community integration installation instructions][15].
+
+### Agent Configuration
+
+1. Edit the `appkeeper.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory.
+   Change the parameter values as follows:
+   * account: your AWS account
+   * integrationToken: The token you got in Setup step.1 above.
```

Comment:
```suggestion
   * `account`: your AWS account
   * `integrationToken`: The token you got in Setup step 1 above.
```
The step for getting the token is now below. What's the real order for doing these steps in? Put these instructions in that order.